app/views/cookies/_action_required.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="govuk-cookie-banner__message govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h2 class="govuk-cookie-banner__heading govuk-heading-m">
+        <%= t('cookies-banner.default.heading') %>
+      </h2>
+      <div class="govuk-cookie-banner__content">
+        <p><%= t('cookies-banner.default.message_1') %></p>
+        <p><%= t('cookies-banner.default.message_2') %></p>
+      </div>
+    </div>
+  </div>
+  <div class="govuk-button-group">
+    <%= button_to t('cookies-banner.button.accept_analytics'),
+        "/cookies/accept_analytics",
```

Comment:
@irisfaraway I've tried to use the URL helper (eg accept_analytics_cookies_path) and get an error `undefined local variable or method `accept_analytics_cookies_path'`. The route is generated correctly - output of `rake routes -c cookies` looks okay. I'm not sure why the URL helpers aren't working - wonder if you have come across this before?

Reply:
That is weird. I had a nose around but nothing was jumping out. Did it only fail with cookies URL helpers or did none of them work?

Reply:
Actually, on further thinking, I think we did run into this issue when using a host app route on a page that came from the engine – maybe in the application layout? It looks like there are some links in there that are just hardcoded rather than using the helper so I think that may be the reason. Seems like there should be a workaround though.

Reply:
Curiously, some of the paths in the engine have `Rails.application.routes.url_helpers` appended, eg. https://github.com/DEFRA/waste-carriers-engine/blob/6e44f3d3dd7db202f5d21abb30683c1d0903df4b/app/views/waste_carriers_engine/edit_cancelled_forms/new.html.erb#L7